package-lock.json

```diff
@@ -1,8 +1,2463 @@
 {
   "name": "discord-bot",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This has been an annoyance for us for a while. Hopefully updating the lockfileVersion to version 2 won't cause unnecessary diffs when people try to submit PRs from setups using version 1. 

That said, even if that happens I think it's pretty low priority at this point since we're trying to get to v13, which uses lockfileVersion 2 anyway (due to upgrade to node 16.x)